app/src/main/res/layout/activity_main.xml

```diff
@@ -21,7 +20,7 @@ xmlns:app="http://schemas.android.com/apk/res-auto">
             android:layout_width="match_parent"
             android:layout_height="match_parent">
 
-            <include layout="@layout/bottom_sheet" />
+            <include layout="@layout/bottom_sheet"/>
```

Comment:
Unnecessary change

Reply:
These changes occur automatically, because I use Android Studio's build-in formatter, using the Ctr+Alt+L key. And that formats the document this way, with all the spaces and extra lines.